main/shortcuts.js

```diff
@@ -18,4 +18,8 @@ function init () {
   // Electron does not support multiple accelerators for a single menu item, so this
   // is registered separately here.
   localShortcut.register('CmdOrCtrl+Shift+F', menu.toggleFullScreen)
+
+  // Control Volume
+  globalShortcut.register('CmdOrCtrl+Up', () => windows.main.send('dispatch', 'changeVolume', 0.1))
+  globalShortcut.register('CmdOrCtrl+Down', () => windows.main.send('dispatch', 'changeVolume', -0.1))
```

Comment:
We should not use `globalShortcut` here. This will register the shortcut at the OS level, even when the app is not focused.

Using `localShortcut` would work, but let's put it in the top menu instead, where all the other shortcuts are.